Sources/FileKit/FileKit.swift

```diff
@@ -82,18 +55,78 @@ public class FileKit {
         return executableURL.appendingPathComponent("..")
         }().standardized
 
-    /// Directory containing the Package.swift of the project (as determined by traversing
+    // MARK: Executable
+
+    /// This URL points to the executable.
```

Comment:
I think it would be worth giving a concrete example of a URL that points to an executable here.

Reply:
Done.